InnerEye/ML/Histopathology/models/deepmil.py

```diff
@@ -242,8 +242,10 @@ def _shared_step(self, batch: Dict, batch_idx: int, stage: str) -> Dict[ResultsK
         predicted_probs = self.activation_fn(bag_logits)
         if self.n_classes > 1:
             predicted_labels = argmax(predicted_probs, dim=1)
+            probs_perclass = predicted_probs
         else:
             predicted_labels = round(predicted_probs)
+            probs_perclass = Tensor([[1.0-predicted_probs[i][0].item(), predicted_probs[i][0].item()] for i in range(len(predicted_probs))])
```

Comment:
PEP8 (space missing around operator)

Reply:
could torch.nn.functional.one_hot be used here for this conversion?

Reply:
These are soft probabilities. I am just mapping the probabilities to vector similar to multi-class case.